test/testdata/lsp/code_actions/sig_missing__parent.rb

```diff
@@ -0,0 +1,10 @@
+# typed: strict
```

Comment:
This file doesn't have any code action assertions, but has an `A` version?

Reply:
Ah, nevermind, it's a multifile test.

Reply:
heh, I got too carried away copying things, clearly.

Reply:
oh wait, no

Reply:
it gets modified by a code action in the other file!